### PR TITLE
solution

### PR DESCRIPTION
**Pull request by OleksandrDzindziura**

### CODE REVIEW

**Review by george-w-boo — APPROVED** [review]
бездоганно 👍

File: src/makeRobotAccountant.js [hunk 1]
```diff
@@ -6,7 +6,17 @@
...
+  let count = 0;
+
+  return (firstNumber) => (secondNumber) => {
+    count += 1;
+
+    if (count > 3 && count % 2 === 0) {
+      return 'Bzzz... Error!';
+    } else {
+      return firstNumber + secondNumber;
+    }
```
Comment (VinnieJ-2k20) [inline src/makeRobotAccountant.js:18]: If we write `return` inside `if` , `else` is redundant.
```suggestion
    if (count > 3 && count % 2 === 0) {
      return 'Bzzz... Error!';
    } 
    
    return firstNumber + secondNumber;
```

File: src/makeRobotAccountant.js [hunk 2]
```diff
@@ -6,7 +6,17 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return (firstNumber) => (secondNumber) => {
```
Comment (VinnieJ-2k20) [inline src/makeRobotAccountant.js:11]: Not the most readable syntax. Fix it.
```suggestion
  return (firstNumber) => {
    return (secondNumber) => {
```

**Review by VinnieJ-2k20 — CHANGES_REQUESTED** [review]
Solution is correct, but need to fix code style.

**Review by VinnieJ-2k20 — APPROVED** [review]
Good job 👍

**Pull request closed without merging**